app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
```

Comment:
giải thích code `&.` chị nghe

Reply:
Dạ &. cũng giống && ạ .

Reply:
Ai nói giống 
`&.` đây là toán tử an toàn của ruby nhé